helix-core/src/main/java/org/apache/helix/manager/zk/ParticipantManager.java

```diff
@@ -240,7 +258,7 @@ private void createLiveInstance() {
      */
     if (retry) {
       try {
-        _zkclient.createEphemeral(liveInstancePath, liveInstance.getRecord());
+        _zkclient.createEphemeral(liveInstancePath, liveInstance.getRecord(), sessionId);
         LOG.info("LiveInstance created, path: " + liveInstancePath + ", sessionId: " + liveInstance
             .getEphemeralOwner());
       } catch (Exception e) {
```

Comment:
The error message is not quite accurate here, it may be caused by  other failures, let us catch NodeExistException separately. For other Exception, print out the exact cause from the exception.